src/main/java/com/mixpanel/android/viewcrawler/EditState.java

```diff
@@ -23,10 +25,15 @@
  */
 /* package */ class EditState extends UIThreadSet<Activity> {
 
-    public EditState() {
+    public interface EditErrorMessage {
```

Comment:
Let's name this `OnErrorListener` and have the callback be named `OnError`. We don't use this convention very well in our code, but it's the platform convention for event listeners like this one.